core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4577,11 +4577,17 @@ public RpcContext createRpcContext() throws UnavailableException {
 
   private LockingScheme createLockingScheme(AlluxioURI path, FileSystemMasterCommonPOptions options,
       LockPattern desiredLockMode) {
+    return createLockingScheme(path, options, desiredLockMode, true);
+  }
+
+  private LockingScheme createLockingScheme(AlluxioURI path, FileSystemMasterCommonPOptions options,
+      LockPattern desiredLockMode, boolean directParentRecursive) {
```

Comment:
can we change the name of the flag to something like `isGetFileInfo` to be more explicit?

Reply:
Yeah, I get it.